packages/backend/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
```

Comment:
I'd say we should save version 1.0.0 for the first release, and this one should be something like 0.1.0 or something. :D